README.md

```diff
@@ -15,10 +15,11 @@ To use the language you need to install the following dependencies:
 - python3
 - pip
 - fasm
+- wabt(required only for the `wat64` target)
+Also we recommend installing the following dependencies for testing purposes:
 - nodejs
-- wabt
 
-On debian/ubuntu based systems with APT you can do:
+To setup directly on your machine:

```

Comment:
This is debian/ubuntu specific. For people on Arch, RHEL, nix,.etc it wont work hence the qualifier.

Reply:
> This is debian/ubuntu specific. For people on Arch, RHEL, nix,.etc it wont work hence the qualifier.

Yes, but the comment right next to the apt command specifies to use your package manager and the only command, that changes depending on your distro, is that apt command. I think specifying it to be debian/ubuntu based specifically might confuse people, who might think the only supported option on other distros is to use the docker container.